src/modules/draft/draftApiInterfaces.ts

```diff
@@ -73,7 +73,8 @@ export type DraftStatusTypes =
   | 'AWAITING_UNPUBLISHING'
   | 'UNPUBLISHED'
   | 'ARCHIVED'
-  | 'AWAITING_ARCHIVING';
+  | 'AWAITING_ARCHIVING'
+  | string;
```

Comment:
Er ikke dette litt skummelt?

Reply:
Jo jo, veldig, og dette her er ikke bra, men det er for å få typescript til å slutte å klage. Typen fra backenden er definert som `string` istedenfor enum som vi har her, og da vil det være mismatch. Dette her fikser jeg nå atm. ved å generere enums fra backenden slik at typen er også enum i frontenden. Så en shitty midlertidig løsning som jeg fikser i den andre branchen med alle typefiksene.

Reply:
Hadde jeg fjernet hele `DraftStatusTypes` måtte jeg da fikse flere typefeil ved det. Og det var det jeg prøvde å unngå for PR'en er myye større enn det jeg ville at den skulle være, nettopp pga typer som måtte justeres på. Så en shitty-quick-fix på `| string` er for å unngå større PR og confusion :sweat_smile: